composer.json

```diff
@@ -21,7 +21,7 @@
         }
     },
     "require-dev": {
-        "php": "^7.0",
+        "php": "^7.1",
```

Comment:
Since the `PHPUnit:^7.5` requires `php-7.1` version at least, it should let this package require `php-7.1` version at least.